gui/gui/src/TGClient.cxx

```diff
@@ -146,8 +146,6 @@ TGClient::TGClient(const char *dpyName)
 
    // Open the connection to the display
    if ((fXfd = gVirtualX->OpenDisplay(dpyName)) < 0) {
-      Error("TGClient", "can't open display \"%s\", switching to batch mode...\n In case you run from a remote ssh session, reconnect with ssh -Y",
-            gVirtualX->DisplayName(dpyName));
```

Comment:
Why do you completely remove the message?

Reply:
See the lengthy commit message in [`ae2f4f1` (#9314)](https://github.com/root-project/root/pull/9314/commits/ae2f4f1b203da94bd44c7933cbbebd7922408f57) - does that answer it?

Reply:
Yes, sorry, I missed it...